manifests/mod/userdir.pp

```diff
@@ -33,6 +36,7 @@
 class apache::mod::userdir (
   $home = undef,
   $dir = undef,
+  $userdir = undef,
```

Comment:
Can we add a data type for this? Is it `Optional[Stdlib::Absolutepath]` or `Optional[String[1]]`?